### ISSUE TITLE
Deprecation warning on Rails 6.0

### ISSUE THREAD

**Issue by cecilian** [thinkerbot/configurable#5]
Hey there,

first of all thank you for this useful little gem.
We're still using it on our Rails 6.0 project and we noticed this warning when using the `decimal` data type:
```
 warning: BigDecimal.new is deprecated; use BigDecimal() method instead
 ```

Would you accept a PR to fix this warning?

**Comment by thinkerbot** [comment]
Hi @cecilian, sure I'm open to a PR!  I'm curious to see what it will look like.  I haven't looked at this code in many years, but browsing it I don't see BigDecimal used anywhere.

**Comment by cecilian** [comment]
Oh well...it was there when I looked at it locally, and in fact the problem is that I went and picked the wrong gem on Github! 

My problem is actually with the similarly named configurable-engine gem and now I can see they released a fix for the warning back in August.

So what can I say, thank you for your prompt reply and sorry for wasting your time. I can at least hope you didn't mind the trip down memory lane :)
